README.md

```diff
@@ -1,64 +1,250 @@
+
+* [`safe_sign_and_submit_transaction`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.safe_sign_and_submit_transaction) — Signs a transaction locally, then submits it to the XRP Ledger. This method does not implement [reliable transaction submission](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission) best practices, so only use it for development or testing purposes.
+
+* [`safe_sign_transaction`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.safe_sign_transaction) — Signs a transaction locally. This method **does  not** submit the transaction to the XRP Ledger.
+
+* [`send_reliable_submission`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.send_reliable_submission) — An implementation of the [reliable transaction submission guidelines](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission), this method signs a transaction locally, submits the transaction to the XRP Ledger, and then verifies that it has been included in a validated ledger (or has failed to do so). Use this method to submit transactions for production purposes.
+
+
+```py
+from xrpl.models.transactions import Payment
+from xrpl.transaction import send_reliable_submission
+from xrpl.ledger import get_latest_validated_ledger_sequence
+
+current_validated_ledger = get_latest_validated_ledger_sequence(client)
+
+# prepare the transaction
+# the amount is expressed in drops, not XRP
+# see https://xrpl.org/basic-data-types.html#specifying-currency-amounts
+my_tx_payment = Payment(
+    account="rMPUKmzmDWEX1tQhzQ8oGFNfAEhnWNFwz",
+    amount="2200000",
+    destination="rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+    last_ledger_sequence=current_validated_ledger + 20,
+    sequence=test_wallet.next_sequence_num,
+    fee="10",
+)
+# sign the transaction
+my_tx_payment_signed = safe_sign_transaction(my_tx_payment,test_wallet)
 
-```bash
-poetry run nox -rs tests
+# submit the transaction
+tx_response = send_reliable_submission(my_tx_payment, test_wallet, client)
 ```
 
-## Running integration tests
+#### Get fee from the XRP Ledger
 
-Integration tests are expensive and often flaky. As a result, they don't run
-by default with the above command. You'll need to currently run integration
-tests manually for each package like so:
+In most cases, you should specify `10` for the `fee` field unless you have a strong reason not to. But if you want to get the [current transaction cost](https://xrpl.org/transaction-cost.html#current-transaction-cost) from the XRP Ledger, you can use the `get_fee` function:
 
-```bash
-poetry run python -m unittest discover tests.integration.transactions
+```py
+from xrpl.ledger import get_fee
+
+FEE = get_fee(client)
+print(FEE)
+
+# print output
+10
 ```
 
-or
+#### Auto-filled fields
```

Comment:
@mvadari added this per your comment. Could you take a quick look before I merge?